packages/components/src/VirtualizedList/CellActions/CellActions.scss

```diff
@@ -1,17 +1,32 @@
 .tc-list-actions {
 	button {
-		height: 20px;
-		line-height: inherit;
 		opacity: 0;
-		padding-left: 0;
 	}
 }
 
 // manage actions display on row hover
-
 :global(.tc-list-large-row) .tc-list-actions :global(.tc-actions) > button,
 :global(.ReactVirtualized__Table__row) .tc-list-actions :global(.tc-actions) > button:focus,
 :global(.ReactVirtualized__Table__row):hover .tc-list-actions :global(.tc-actions) > button {
 	opacity: 1;
 	transition: opacity 0.15s ease-in;
 }
+
+// Large display specificities
+:global(.tc-list-large-row) .tc-list-actions :global(.tc-actions) > button {
```

Comment:
In fact we don't have a choice because I want to style the cell within the cell's css.
This selector is to scope to a specific display.

Why : 
* an add/modification on cell won't require to touch to other parts
* the style stays just next to the cell that is styled, resier to search (same concept as the tests)